<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="idLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lastNameLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="firstNameLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="middleNameLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="addressLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="phoneNumberLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="medicalCardLineEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="diagnosisLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="createObjectButton">
      <property name="text">
       <string>Create Object</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="showButton">
      <property name="text">
       <string>Show</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
